packages/cypress-commands/src/commands.ts

```diff
@@ -74,7 +74,7 @@ declare global {
        * cy.get('[ui5-list]').clickUi5ListItemByText("List Item")
        * cy.clickUi5ListItemByText("List Item")
        */
-      clickUi5ListItemByText(text: string, options: Partial<ClickOptions>): Chainable<Element>;
+      clickUi5ListItemByText(text: string, options?: Partial<ClickOptions>): Chainable<Element>;
```

Comment:
`{force: true}` is not supported by this custom command:

```suggestion
      clickUi5ListItemByText(text: string, options?: Partial<Omit<ClickOptions, 'force'>>): Chainable<Element>;
```

Could you please also add a note to the jsdoc `@param options` annotation. E.g.:

```
@param [options] ClickOptions (without `force`)
```

Reply:
Code is updated.
I changed the comment to 
```
* @param {Partial<ClickOptions>} [options] ClickOptions (without `force`)
```
Since storybook is failed to parse the type from the code.
it's not very accurate but I think it should be enough 😅